pantheon-bundle/src/main/java/com/redhat/pantheon/auth/user/AuthUserInfoServlet.java

```diff
@@ -0,0 +1,72 @@
+package com.redhat.pantheon.auth.user;
+
+import com.google.common.collect.Lists;
+import com.redhat.pantheon.servlet.ServletUtils;
+import org.apache.jackrabbit.api.security.user.Group;
+import org.apache.jackrabbit.api.security.user.User;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.apache.sling.servlets.annotations.SlingServletPaths;
+import org.jetbrains.annotations.NotNull;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Component;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.collect.Maps.newHashMap;
+
+/**
+ * Custom api to render Auth User info.
+ * It provides:
+ *   userId: default value "anonymous"
+ *   groups: default value "[]"
+ *   userType: default value ""
+ *
+ * @author Lisa Davidson
+ */
+@Component(
+        service = Servlet.class,
+        property = {
+                Constants.SERVICE_DESCRIPTION + "=Servlet to fetch the current build date in the Admin Panel",
```

Comment:
minor: update the service description for this servlet

Reply:
done